src/components/chips/chips-theme.scss

```diff
@@ -15,6 +15,11 @@ md-chips.md-THEME_NAME-theme {
         color: '{{primary-contrast}}';
       }
     }
+
+    &.md-chip-editing {
+      background: #fff;
```

Comment:
Should this color be hard-coded? Is there a variable for the general chipbar background color?